scripts/pyqt5_to_pyqt6/pyqt5_to_pyqt6.py

```diff
@@ -109,6 +110,7 @@
     QtXml,
     QtNetwork,
     QtPrintSupport,
+    QtQuickWidgets,
```

Comment:
Shall we not add it above in the from `PyQt6 import (`  list to be able to use it here ?

Reply:
oops :)